validation/jinja2/validation/_base.html

```diff
@@ -14,12 +15,15 @@
 const csrftoken = document.querySelector('[name=csrfmiddlewaretoken]').value;
 </script>
       <script src="/static/index.js"></script>
+      <script src="/static/menu.js"></script>
```

Comment:
I don't understand what's going on with the static stuff, but this is the _cromulent_ way of including static files:

```suggestion
      <script src="{% static 'menu.js' }%"></script>
```

Please see: https://docs.djangoproject.com/en/3.2/howto/static-files/#configuring-static-files